src/wp-includes/class-wp-plugin-dependencies.php

```diff
@@ -369,10 +369,29 @@ public static function get_dependency_data( $slug ) {
 	 */
 	public static function display_admin_notice_for_unmet_dependencies() {
 		if ( in_array( false, self::get_dependency_filepaths(), true ) ) {
+			$error_message = __( 'Some required plugins are missing or inactive.' );
+
+			if ( is_multisite() ) {
+				if ( current_user_can( 'manage_network_plugins' ) ) {
+					$error_message .= ' ' . sprintf(
+						/* translators: %s: Link to the network plugins page. */
+						esc_url( network_admin_url( 'plugins.php' ) )
+					);
```

Comment:
This is missing the string in which the URL is placed.

It would be good to check with an i18n expert that the string can be concatenated like this and still be translated. If not we might need a few more if statements. @swissspidy Are you able to help advise here?

Reply:
Good catch. I fixed the missing string.

Good call out on the concatenation. We did [confirm this is acceptable on the other PR](https://github.com/WordPress/wordpress-develop/pull/6074#discussion_r1511837905).

Reply:
> Good call out on the concatenation. We did confirm....

Thanks. As a monolingual English speaker I like to check these things because it's easy for me to misunderstand.